### PR TITLE
Update volts2

### PR DESCRIPTION
**Pull request by JebGy**
I add my Theme :3

### CODE REVIEW

**Comment by dzhou121** [comment]
Thanks for the theme. Can you please put it to the end of the json?

**Comment by JebGy** [comment]
> Thanks for the theme. Can you please put it to the end of the json?

That's it 👍👍

File: volts2 [hunk 1]
```diff
@@ -238,5 +238,13 @@
         "author": "Everblush",
         "description": "A Dark, Vibrant and Beatiful colorscheme",
         "meta": "https://raw.githubusercontent.com/Everblush/lapce/main/volt.toml"
+    },
+    {
+        name = "jeb-theme"
```
Comment (dzhou121) [inline volts2:243]: They are not in json format.

Comment (JebGy) [reply volts2:243]: let me see

Comment (JebGy) [reply volts2]: Now that's it, sorry for mistakes, because is my first time 😢😢

File: volts2 [hunk 2]
```diff
@@ -238,5 +238,13 @@
         "author": "Everblush",
         "description": "A Dark, Vibrant and Beatiful colorscheme",
         "meta": "https://raw.githubusercontent.com/Everblush/lapce/main/volt.toml"
+    },
+    {
+        "name" = "jeb-theme",
```
Comment (dzhou121) [inline volts2:243]: ```=``` should be ```:``` as well.

Comment (JebGy) [reply volts2:243]: Oh, ok let me see XD

Comment (JebGy) [reply volts2]: Now?

**Pull request merged**